e2e/features/request-hearing-requirements.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Request hearing requirements
+
+  Background:
+    Given I am signed in as a `Legal Rep`
+    And I create a new case
+    And I save my initial appeal
+    And I submit my appeal
+    And I switch to be a `Case Officer`
+    And I request respondent evidence
+    And I upload respondent evidence
+    And I switch to be a `Legal Rep`
+    And I build my case
+    And I submit my case
+    And I switch to be a `Case Officer`
+    And I request respondent review
+    And I add the appeal response
+
+  @create-direction @RIA-385
+  Scenario: Request respondent review
+
+    When I select the `Request hearing requirements` Next step
+    Then I am on the `Request hearing requirements` page
+    When I click the `Send direction` button
+    Then I should see an alert confirming the case `has been updated with event: Request hearing requirements`
```

Comment:
can we get a check for the new State in CCD too

Reply:
i.e. "Submit hearing requirements" as per the ACs

Reply:
Have discussed with @fantasticjamieburns that this would be better covered by a unit test once the JIRA for CCD definition unit tests has been played.